box-content-sdk/src/main/java/com/box/androidsdk/content/requests/BoxHttpResponse.java

```diff
@@ -198,7 +202,7 @@ private String readStream(InputStream inputStream) throws IOException, BoxExcept
                 builder.append(buffer, 0, read);
                 read = reader.read(buffer, 0, BUFFER_SIZE);
             }
-
+            disconnect();
```

Comment:
This cannot be called after reader.close the code is essentially inherited from:
https://github.com/box/box-java-sdk/blob/94ce2b691b7faa47835367bc55b371e62b779005/src/main/java/com/box/sdk/BoxJSONResponse.java